testing/src/test/java/io/stargate/it/storage/ExternalStorage.java

```diff
@@ -32,13 +33,15 @@
 import org.slf4j.LoggerFactory;
 
 /** JUnit 5 extension for tests that need a backend database cluster managed by {@code ccm}. */
-public class ExternalStorage implements BeforeAllCallback, AfterAllCallback, ParameterResolver {
+public class ExternalStorage
+    implements BeforeAllCallback, AfterAllCallback, ParameterResolver, BeforeTestExecutionCallback {
 
   private static final Logger LOG = LoggerFactory.getLogger(ExternalStorage.class);
 
   private static final String CCM_VERSION = "ccm.version";
   private static final boolean EXTERNAL_BACKEND =
       Boolean.getBoolean("stargate.test.backend.use.external");
+  private static final String DATACENTER = System.getProperty("stargate.test.backend.dc", "dc1");
```

Comment:
We need to set this somewhere when not using DSE.
See `PersistenceTest.querySystemTables()`, it currently fails for Cassandra 3.11.

Reply:
Yes, addressed in 1ce9bd0 by not using simple snitch